pkg/system/phase2_creating.go

```diff
@@ -1239,6 +1235,15 @@ func (r *Reconciler) SetDesiredCoreAppConfig() error {
 		"DISABLE_DEV_RANDOM_SEED":    "true",
 		"NOOBAA_LOG_LEVEL":           "default_level",
 	}
+	// check if provider mode is enabled and signal the core
+	annotationValue, annotationExists := util.GetAnnotationValue(r.NooBaa.Annotations, "MulticloudObjectGatewayProviderMode")
+	annotationBoolVal := false
+	if annotationExists {
+		annotationBoolVal = strings.ToLower(annotationValue) == strconv.FormatBool(true)
```

Comment:
Why is `strconv.FormatBool` necessary?

Reply:
to prevent typos at the very least, i try to avoid hardcoding text strings if not necessary